test/integration/WebSocketsProtocol.test.ts

```diff
@@ -12,9 +12,10 @@ describe('A server with the Solid WebSockets API', (): void => {
 
   beforeAll(async(): Promise<void> => {
     const factory = await instantiateFromConfig(
-      'urn:solid-server:default:ServerFactory', 'websockets.json', {
+      'urn:solid-server:default:ServerFactory', 'allow-auth.json', {
```

Comment:
Took me a while to understand the filename 😄
`no-authorization.json` or `auth-allow-all.json`?